jobs/JREGIONAL_RUN_ANAL

```diff
@@ -74,7 +74,7 @@ mkdir_vrfy -p ${ANALWORKDIR}
 $SCRIPTSDIR/exregional_run_analysis.sh \
              CYCLE_DIR="${CYCLE_DIR}" \
              ANALWORKDIR="${ANALWORKDIR}" \
-             FG_ROOT="${FG_ROOT}" || \
+             FG_ROOT="${CYCLE_ROOT}" || \
```

Comment:
This should be "FG_ROOT="${FG_ROOT}"

Reply:
GSI analysis will find the background in fcst_fv3lam/INPUT, which is created by task: PREP_WARMSTART_TN.

So task ANAL_GSI_TN has no "FG_ROOT". It only has CYCLE_ROOT. 

Please check FV3LAM_wflow.xml for this section.

Reply:
Okay I see. Those codes for RTMA to find its FG were moved to PREP_WARMSTART_TN and FG_ROOT is no longer required by ANAL_GSI_TN. In this situation, would it be better to remove FG_ROOT for the ANAL jjob and script? But it is okay to leave as is.